src/components/LayerPanel.vue

```diff
@@ -38,28 +27,17 @@
 </template>
 
 <script>
+import BreachControl from '@/components/BreachControl'
 import LayerControl from '@/components/LayerControl'
+import LayerPanelItem from '@/components/LayerPanelItem'
+
 
 export default {
   props: {
-    items: {
+    layerSets: {
```

Comment:
Leaflet refers to `layers` and `LayerGroups`,  here we use  `layerSets`. Which terminology do we use?